### DISCUSSION TITLE
Advanced Search Accessibility

### DISCUSSION THREAD

**Discussion by ObaltzerS** [PFAS-Project/governance-db-search-tool#5] [General]
Hey just doing a little work today, but I noticed a pretty big issue with the current design of the site. If all of the advanced search options are expanded, the majority of them are inaccessible because the menu is not scrollable.

**Comment by ProfJanetDavis** [comment]
It looks like you made those into popup menus? If we do that, we should
show a summary of the filters, since users want to see what they are
searching for.

Alternatively, we could go back to the older design and make it scrollable.

On Mon, Jul 3, 2023 at 11:34 AM Oliver Baltzer ***@***.***>
wrote:

> Hey just doing a little work today, but I noticed a pretty big issue with
> the current design of the site. If all of the advanced search options are
> expanded, the majority of them are inaccessible because the menu is not
> scrollable.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/PFAS-Project/discussions/5>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEXJIU6QULEVZ2GYWAI6TATXOMGCZANCNFSM6AAAAAAZ4ZBWS4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Janet Davis (she/her)
Department Chair, Computer Science
Associate Professor and Microsoft Chair of Computer Science
Whitman College, Walla Walla, WA 99362
206-383-8798 (cell), http://cs.whitman.edu/~davisj/
Schedule an appointment: http://meetme.so/JanetDavis

**Reply by ObaltzerS** [reply]
I made them into popup menus so that I could more easily create the filters without having to mess with the scrollability, though I am not entirely tied to it. I did prefer how the older design looked, and the filters would likely work the same if we went that way and decide to implement the scrollability.

**Comment by ProfJanetDavis** [comment]
I also preferred how the older design looked. Should we ask Molly to let
the left sidebar scroll with the search results?

On Fri, Jul 7, 2023 at 2:53 PM Oliver Baltzer ***@***.***>
wrote:

> I made them into popup menus so that I could more easily create the
> filters without having to mess with the scrollability, though I am not
> entirely tied to it. I did prefer how the older design looked, and the
> filters would likely work the same if we went that way and decide to
> implement the scrollability.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/PFAS-Project/discussions/5#discussioncomment-6388490>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEXJIUZB5OS7HY2RJWRW6R3XPCAM5ANCNFSM6AAAAAAZ4ZBWS4>
> .
> You are receiving this because you commented.Message ID:
> <PFAS-Project/governance-db-search-tool/repo-discussions/5/comments/6388490
> @github.com>
>


-- 
Janet Davis (she/her)
Department Chair, Computer Science
Associate Professor and Microsoft Chair of Computer Science
Whitman College, Walla Walla, WA 99362
206-383-8798 (cell), http://cs.whitman.edu/~davisj/
Schedule an appointment: http://meetme.so/JanetDavis
